src/quo2/components/inputs/browser_input/style.cljs

```diff
@@ -0,0 +1,52 @@
+(ns quo2.components.inputs.browser-input.style
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]))
+
+(defn clear-icon
+  [blur? override-theme]
+  (if blur?
+    (colors/theme-colors colors/neutral-80-opa-30 colors/white-opa-10 override-theme)
+    (colors/theme-colors colors/neutral-40 colors/neutral-60 override-theme)))
```

Comment:
we should only define styles on the style.cljs file, you can move this to view file

Reply:
Alright